wrapper.go

```diff
@@ -130,3 +132,27 @@ func createExporter(printStdout bool, ctx context.Context, logger log.FieldLogge
 	}
 	return newExporter(ctx, logger)
 }
+
+func recoverAndCheckFailWriteSpan() {
+	defer recoverWithLogs()
+	dirEntries, err := os.ReadDir(SPANS_DIR)
+	if err != nil {
+		logger.WithError(err).Error("failed to read spans dir")
+	}
+	found := false
+	for _, file := range dirEntries {
+		if !file.IsDir() {
+			if strings.Contains(file.Name(), "_end") {
+				found = true
+				break
+			}
+		}
+	}
+	if !found {
+		endFile, err := os.Create(filepath.Join(SPANS_DIR, "balagan_end"))
+		if err != nil {
+			logger.WithError(err).Error("failed to create file _end in spans dir lambda will timeout :/")
+		}
+		endFile.Close()
```

Comment:
IMO, you should have a new "global member" of the tracer, that says that we're currently inside a malformed state,
the extension is not listening to us anymore, and we shouldn't do anything.
WDYT?

Reply:
env var?

Reply:
env var is great 👌

Reply:
Why is env var better? Won't it be more complicated?
Also, we collect the env vars so we need to verify that we don't collect this env var.